addons/im_livechat/static/src/js/im_livechat_channel_form_controller.js

```diff
@@ -0,0 +1,43 @@
+odoo.define('im_livechat.ImLivechatChannelFormController', function (require) {
+"use strict";
+
+const FormController = require('web.FormController');
+
+let ImLivechatChannelFormController = FormController.extend({
+    events: Object.assign({}, FormController.prototype.events, {
+        'click .o_im_livechat_channel_form_reset1': '_onResetColors1',
+        'click .o_im_livechat_channel_form_reset2': '_onResetColors2',
+    }),
+    /**
+     * @private
+     */
+    _onResetColors1: function(){
+        this.$('[name="button_background_color"] > .o_field_color').css('background-color', "#878787");
```

Comment:
I would remove the `>`, this is a detail and some layout changes would require changing this too.